soccer/planning/planner/PivotPathPlanner.cpp

```diff
@@ -71,24 +75,33 @@ Trajectory PivotPathPlanner::plan(const PlanRequest& request) {
         points.push_back(point);
     }
 
-    BezierPath pathBezier(points, start_instant.linear_velocity(), Point(0, 0),
+    BezierPath pathBezier(points, Point(0, 0), Point(0, 0),
```

Comment:
This feels a little weird. What's the reasoning behind this?

Reply:
For pivot, you actually want to explicitly create a circular(ish) path rather than respecting initial velocity.

Without this, when you're constantly replanning the robot just sort of pushes the ball forwards forever.

Reply:
That makes sense. How well does it work with a fast moving robot coming into a pivot or a robot far away from the pivot?

Reply:
It's a little jank when the robot comes in fast, but it actually gives decent results. It definitely needs to be fixed eventually, if we want to tune motion control really aggressively (which we do). No idea how it works if the robot is very far away from the pivot - with the transition to the action/behavior controller (hey, that's ABC!) we should honestly just fail if we attempt to do a pivot in that context.

Reply:
Sounds good! So the idea would be to move close to a ball/have it captured, before calling this?